From 9eb9a4e8bceebb5324e172093693efb64ac0dbbd Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Wed, 18 Dec 2019 00:19:57 -0800
Subject: [PATCH] plugins/coreboot: Add missing fu_hash dependency

This fixes the build when using samurai in place of ninja.
---
 plugins/coreboot/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plugins/coreboot/meson.build b/plugins/coreboot/meson.build
index 2ec0270f..80480f8c 100644
--- a/plugins/coreboot/meson.build
+++ b/plugins/coreboot/meson.build
@@ -5,6 +5,7 @@ install_data(['coreboot.quirk'],
 )
 
 shared_module('fu_plugin_coreboot',
+  fu_hash,
   sources : [
     'fu-plugin-coreboot.c',
     'fu-coreboot-common.c',
-- 
2.24.1

